server/spinwick.go

```diff
@@ -584,7 +584,15 @@ func checkMMPing(ctx context.Context, client *mattermostModel.Client4) error {
 }
 
 func makeSpinWickID(repoName string, prNumber int) string {
-	return strings.ToLower(fmt.Sprintf("%s-pr-%d", repoName, prNumber))
+	domainName := ".test.mattermost.cloud"
```

Comment:
you can use `s.Config.DNSNameTestServer` this have this config setting

Reply:
Good point! Changed.